extensions/resource-deployment/src/interfaces.ts

```diff
@@ -98,6 +98,8 @@ export interface WizardInfo {
 
 export interface NotebookBasedDialogInfo extends DialogInfoBase {
 	notebook: string | NotebookInfo;
+	runNotebook?: boolean;
```

Comment:
if optional then should they be suffixed with ?

Reply:
Is this a question or change request?, i am not following.

Reply:
suggestion as you might have a good reason to not include ? so you could respond with that.

Reply:
suggestion :)

Reply:
this is optional and i already have the ? character.

Reply:
Ah ok, i should change my color scheme, it makes some characters hard to see.